service/src/lib.rs

```diff
@@ -71,8 +71,7 @@ impl<Config: Clone + Send + 'static, S: Service<Config>> ConnectionManager<Confi
             let config = self.config.clone();
             let (shutdown_tx, _) = tokio::sync::broadcast::channel(16);
 
-            let bitcoin_core = self.bitcoin_config.new_client(None)?;
-            bitcoin_core.connect().await?;
+            let bitcoin_core = self.bitcoin_config.new_client(None).await?;
```

Comment:
why not connect here?

Reply:
`connect` is now bundled in `new_client` to fetch the network